lib/conjur/base.rb

```diff
@@ -141,6 +141,24 @@ def new_from_key(username, api_key, remote_ip = nil)
       def new_from_token(token, remote_ip = nil)
         self.new.init_from_token token, remote_ip
       end
+
+      # Create a new {Conjur::API} instance from a file containing a token issued by the
+      # {http://developer.conjur.net/reference/services/authentication Conjur authentication service}
+      #
+      # This method is useful when an external process, such as a sidecar container, is continuously
+      # obtaining fresh tokens and writing them to a known file.
+      #
+      # Note that Conjur tokens are issued as JSON.  This method expects the token file to contain JSON.
+      # When sending tokens as headers, you will normally use base64 encoded strings.  Authorization headers
+      # used by Conjur have the form `'Token token="#{b64encode token.to_json}"'`, but this format is in no way
+      # required.
```

Comment:
This paragraph, on the other hand, is too verbose. The part about base64 encoded tokens in particular is of no concern at this level of abstraction; and the last part frankly I'm not sure I understand at all (what do you mean, "this format is in no way required"?).

I'd shorten it to just the first two sentences, just to state that JSON is expected.